ci/appveyor.yml

```diff
@@ -11,12 +11,15 @@ environment:
 platform:
   - x64
 install:
-  - ps: .\appveyor.ps1
+  - ps: .\ci\hardwareinfo.ps1
+  - ps: .\ci\appveyor.ps1
   - npm install
 build: off
 test_script:
   - "npm run %ci_type%"
+on_finish:
+  - ps: .\ci\hardwareinfo.ps1
 cache:
   - node_modules -> package.json
   - C:\Users\appveyor\.ccm\repository
```

Comment:
In theory we would never want to invalidate the .ccm\repository directory, since the cache is per matrix configuration, which includes the full cassandra version, which shouldn't change.  However if things get broken or we fundamentally change how C* is installed, it might be nice to invalidate this anyways.